### DISCUSSION TITLE
Pin docker to OpenMVS 2 + release prebuilt binaries for linux/mac

### DISCUSSION THREAD

**Discussion by jgilat** [cdcseacave/openMVS#1170] [Ideas]
Hey, 

Due to the difficulty of building this binary (or finding prebuilt binaries) for linux/mac, I would really really appreciate if someone either helps with getting to docker images pinned to major 2 or release prebuilt binaries for linux/mac. Thanks!!!

**Comment by jared-krauss** [comment]
same
